processor/redactionprocessor/internal/url/classifier.json

```diff
@@ -0,0 +1 @@
289199017266808,-8.379003345982191,-8.030696651713976,-8.46601472297182,-5.258107929474194,-8.561324902776146,-3.1228109057348257],[-1.596798460957614,-7.6001421705956735,-7.640964165115928,-5.344648685135478,-1.8921021056868312,-6.839336341561913,-8.110967794361665,-1.6228272590921338,-1.7628783045649916,-8.87310784640856,-7.001305669506969,-5.467159861987808,-8.179960665848615,-3.2183655671770013,-2.524218636471301,-8.293289351155618,-9.209580083029774,-4.57000847032435,-4.2425484264156506,-5.65423202154036,-7.560921457442392,-9.209580083029774,-7.307472556632853,-9.209580083029774,-6.877436187794184,-9.02725852623582,-2.172728230713227],[-2.254780968033424,-6.898209866138606,-2.027603216646053,-6.492744758030441,-2.462642464536694,-5.833499129146177,-6.898209866138606,-4.310445830910898,-2.0731012597852527,-6.898209866138606,-6.898209866138606,-6.42820623689287,-6.715888309344651,-6.898209866138606,-4.4558628307694015,-1.4955324842663262,-6.3675816150764355,-4.636446767664815,-5.766807754647505,-1.8703897472882491,-4.37248122183035,-4.348764695213034,-6.802899686334281,-4.473407140420311,-5.175443268397502,-6.898209866138606,-2.5557039896270073],[-3.872874353532438,-6.07983833755968,-5.605956228985375,-6.1495716755743555,-2.9181087352006836,-5.833140189836589,-6.998723104610883,-7.317176835729417,-3.8266394412494407,-8.857621876676566,-7.45070822835394,-4.745382824577915,-4.447858487031085,-5.448125692199715,-2.2255103117197566,-4.737230605516364,-9.03994343347052,-5.751541545953709,-3.132676045163294,-4.230201081753655,-7.038463433260397,-7.65364907235063,-6.230540738108023,-7.70494236673818,-8.16447469611662,-7.731610613820342,-0.3817890191036424],[-2.531179599331457,-6.00314605188182,-5.907835872077495,-4.694813232231641,-0.9292230185496455,-5.907835872077495,-6.00314605188182,-3.199785670975285,-2.3395844057521735,-6.00314605188182,-5.666673815260607,-3.8868905370792675,-4.0016660516716955,-5.009894278871537,-1.7362497244615698,-6.00314605188182,-6.00314605188182,-6.00314605188182,-5.820824495087865,-5.907835872077495,-3.118345339035111,-5.666673815260607,-5.907835872077495,-6.00314605188182,-4.568061526592498,-3.7731316517226094,-3.1699327078256037],[-2.154456318300654,-3.132028909232904,-3.2042402732214343,-3.554775966080049,-3.8320798875631166,-3.2625149066892174,-4.1318426237305275,-2.7847122791384975,-2.7534204117779435,-5.68607428491371,-5.271518205104294,-3.779792629468334,-3.3524684982622075,-3.8123859357235683,-2.644532475328425,-3.3676057306367433,-6.237872536056438,-3.6809218434868227,-2.7030074975159986,-1.86142386740762,-4.4723885304095425,-4.918696830481552,-2.8042850405884527,-7.783949528282384,-4.702039558487341,-8.486442571260568,-3.2910629924454486]],"Positions":{"100":3,"101":4,"102":5,"103":6,"104":7,"105":8,"106":9,"107":10,"108":11,"109":12,"110":13,"111":14,"112":15,"113":16,"114":17,"115":18,"116":19,"117":20,"118":21,"119":22,"120":23,"121":24,"122":25,"32":26,"97":0,"98":1,"99":2},"Threshold":0.018782003473122023}
```

Comment:
you may wanted to use the updated model from here: https://github.com/grafana/clusterurl/blob/main/clusterurl/model.json

It was re-trained to include more HTTP/TCP related vocabulary, see https://github.com/jcreixell/go-gibberish/commit/fa3244dc1cc9cf3f319c85a77d3b31a664e5bb4f

this is particularly useful for next.js applications, which typically send span names with the form `<HTTP_VERB> <url>` (it can be easily reproduced with the OTEL demo)